### PR TITLE
dApp: Integrating Cypress

### PR DESCRIPTION
**Pull request by taleldayekh**
**Thank you for submitting this pull request :)**

Fixes # 

**Short description**

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-dapp/tests/e2e/specs/accept-disclaimer.spec.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-    cy.contains('Disclaimer').should('be.visible');
-    cy.get('.disclaimer__accept-checkbox').click();
-    cy.get('.disclaimer__persist-checkbox').click();
-    cy.get('.disclaimer__accept-button').click();
-    cy.url().should('include', '/#/home');
-  });
+import { disclaimerRoute } from '../fixtures';
+
+it('accepts disclaimer and redirects to home route', () => {
+  disclaimerRoute();
```
Comment (weilbith) [inline raiden-dapp/tests/e2e/specs/accept-disclaimer.spec.ts:4]: Could you add a verb to that function name? At the moment it is like "**What** disclaimer route?".
But very good approach to introduce this! :+1:

**Review by weilbith — APPROVED** [review]
Thanks for doing this first step! Let's keep them small and take it as it is. I think that is pretty good to continue on top of that. Maybe just my small comment there. :wink:

**Comment by weilbith** [comment]
@taleldayekh can you convert this into an actual PR?

**Comment by weilbith** [comment]
I'm not sure if we use fixtures as Cypress defines them.

**Review by weilbith — CHANGES_REQUESTED** [review]
See comments above.

**Comment by taleldayekh** [comment]
Hi think we should wait with this and keep it experimental for now until things are working as we want. It is still a lot of trying-things-out coding, or what do you say?

**Comment by weilbith** [comment]
Hmm. I would like to take small steps but take them. e.g. I would then like to also merge the PR for the first CI integration. This is also not perfect, since e.g. it does not use the integration image yet. We need to figure out a lot of things, but I think they appear on the road and we can't know/tackle them all on the start of that road.
Anyways I would still like to avoid this clashing of the terminology with the fixtures.

File: raiden-dapp/tests/e2e/specs/accept-disclaimer.spec.ts (hunk 1 above)
Comment (taleldayekh) [reply raiden-dapp/tests/e2e/specs/accept-disclaimer.spec.ts]: Done!

**Review by weilbith — APPROVED** [review]
Great :muscle:

**Pull request merged**